tests/test_kvxopt.py

```diff
@@ -0,0 +1,105 @@
+# #!/usr/bin/env python
```

Comment:
The test fixture is commented out?

All CI tests passed except for the linter issue (pointed out in another comment). Maybe we can try uncommenting it now?